e2e/process-services/comment_component_processes.e2e.ts

```diff
@@ -0,0 +1,157 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import LoginPage = require('../pages/adf/loginPage');
+import ProcessServicesPage = require('../pages/adf/process_services/processServicesPage');
+import ProcessFiltersPage = require('../pages/adf/process_services/processFiltersPage.js');
+import { CommentsPage } from '../pages/adf/commentsPage';
+
+
+import TestConfig = require('../test.config');
+import resources = require('../util/resources');
+import Util = require('../util/util.js');
+
+import AlfrescoApi = require('alfresco-js-api-node');
+import { UsersActions } from '../actions/users.actions';
+import { AppsActions } from '../actions/APS/apps.actions';
+
+describe('Comment component for Processes', () => {
+
+    let loginPage = new LoginPage();
+    let processServicesPage = new ProcessServicesPage();
+    let processFiltersPage = new ProcessFiltersPage();
+    let commentsPage = new CommentsPage();
+
+    let app = resources.Files.SIMPLE_APP_WITH_USER_FORM;
+    let tenantId, appId, processInstanceId, comment, taskComment, addedComment;
+
+    beforeAll(async(done) => {
+       this.alfrescoJsApi = new AlfrescoApi({
+            provider: 'BPM',
+            hostBpm: TestConfig.adf.url
+        });
+
+        done();
+    });
+
+    beforeEach(async(done) =>{
+        let apps = new AppsActions();
+        let users = new UsersActions();
+
+        await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        let user = await users.createTenantAndUser(this.alfrescoJsApi);
+
+        tenantId = user.tenantId;
+
+        await this.alfrescoJsApi.login(user.email, user.password);
+
+        let importedApp = await apps.importPublishDeployApp(this.alfrescoJsApi, app.file_location);
+        appId = importedApp.id;
+
+        let processWithComment = await apps.startProcess(this.alfrescoJsApi, 'Task App', 'Comment APS');
+        processInstanceId = processWithComment.id;
+
+        await loginPage.loginToProcessServicesUsingUserModel(user);
+
+        done();
+    });
+
+    afterEach(async(done) => {
+        await this.alfrescoJsApi.activiti.modelsApi.deleteModel(appId);
```

Comment:
It would be better that you try to login before and then try to delete the model